service-templates/load-balanced-fargate-svc/v1/instance_infrastructure/main.tf

```diff
@@ -1,6 +1,5 @@
 resource "aws_security_group" "lb_sg" {
   count       = var.service_instance.inputs.loadbalancer_type == "application" ? 1 : 0
-  name        = "service_lb_security_group"
```

Comment:
We don't want the name anymore? Change does not seem related to "Components" is it?

Reply:
This was a general issue I came across while testing the template - providing a fixed name for the security group means you can only have one instance of this template per account. I think we should omit resource names whenever it's supported.